build-tools/validate-e2e

```diff
@@ -10,7 +10,7 @@ pids=""
 trap 'kill $pids' EXIT
 
 export PORT=5050
-env HEROKU_APP_NAME=localhost SECRET_KEY=TheSecret python app.py > e2e-server.log 2>&1 &
+env HEROKU_APP_NAME=localhost BASE_URL=http://localhost:$PORT SECRET_KEY=TheSecret python app.py > e2e-server.log 2>&1 &
```

Comment:
Hi @rix0rrr ! I just added the `BASE_URL` env variable to the script that runs the E2E tests. We need that variable now on a couple places other than the email sending. Let me know if this looks right to you as well.